decrypt.js

```diff
@@ -0,0 +1,12 @@
+const NodeRSA = require('node-rsa')
+
+const rsa = new NodeRSA().importKey(process.env.RSA_PRIVATE_KEY)
+
+module.exports = encryptData => {
```

Comment:
small improvement that make it easier to understand ;)

```suggestion
module.exports = encryptedData => {
```